startup/26-pump_ultra.py

```diff
@@ -0,0 +1,492 @@
+from ophyd import Device, EpicsMotor, EpicsSignal, EpicsSignalRO
+from ophyd import Component as Cpt
+import time
+import pandas as pd
+import numpy as np
+
+class syrng_ultra(Device):
```

Comment:
Class name should follow the PascalCase convention.
```suggestion
class SyrngUltra(Device):
```
Please update accordingly below, when the instantiation happens.